package.json

```diff
@@ -8,6 +8,10 @@
     "email": "justin@mecham.me",
     "url": "http://justin.mecham.me"
   },
+  "private": true,
```

Comment:
Yep, this allows on APM but not on NPM. It's a safeguard because too many people have published Atom packages to NPM accidentally